### PR TITLE
extending TxPaginationMeta

### PR DESCRIPTION
**Pull request by Pawlak00**
This is an attempt to add possibility of using extended TxPaginaitonMeta

### CODE REVIEW

**Comment by baziorek** [comment]
The issue depends on features from https://github.com/hyperledger/iroha/pull/1092

**Review by baziorek — COMMENTED** [review]
I've checked changes and found few things to change.
Unfortunately I'm not JS specialist.

For now please after changing my suggestions make the PR not draft.

File: example/chain.ts [hunk 1]
```diff
@@ -45,3 +53,30 @@ new BatchBuilder([
   .send(commandService)
   .then(res => console.log(res))
   .catch(err => console.error(err))
+// set timestamps to correct values
+const firstTxTime = new Timestamp()
+firstTxTime.fromDate(new Date(1626342627382))
```
[the remark this answers is not in the document]
Comment (Pawlak00) [reply example/chain.ts:58]: I decided to use height in example query, it will be easier

**Review by baziorek — APPROVED** [review]
Everything looks good for me, but I'm not JS programmer.

**Review by stefashkaa — APPROVED** [review]

**Pull request merged**